runtime/queries/lean/locals.scm

```diff
@@ -0,0 +1,3 @@
+(module) @scope
+(namespace) @scope
+(section) @scope
```

Comment:
```suggestion
[
  (module)
  (namespace)
  (section)
] @local.scope
```

nvim does locals a tiny bit differently as well

other than this, this PR looks good to me! :slightly_smiling_face: